src/PlaywrightSharp.Tests/Page/GoToTests.cs

```diff
@@ -533,15 +549,18 @@ public async Task ShouldOverrideReferrerPolicy()
             Server.SetRoute("/grid.html", context =>
             {
                 context.Response.Headers["Referrer-Policy"] = "no-referrer";
-                return Task.CompletedTask;
+                string file = File.ReadAllText(Path.Combine(PlaywrightSharpLoader.ContentRoot, "wwwroot", "grid.html"));

```

Comment:
not sure about a clean way for this

Reply:
I implemented a Subscribe method here https://github.com/hardkoded/playwright-sharp/blob/a958f565cf46db5b3e7739df27970ed132844884/src/PlaywrightSharp.Tests/Webkit/ProvisionalTests.cs#L32

You add the `Referrer-Policy` on a `Subscribe` and then let the server handle the file.